### PR TITLE
Fix logo widths

### PR DESCRIPTION
**Pull request by FrancisBehnen**
Should fix the issue on Windows, while still wrapping in Safari

### CODE REVIEW

**Comment by FrancisBehnen** [comment]
Anyone strong feelings about these `30px`s? Otherwise we're merging this.

**Comment by skamoen** [comment]
Can't test on Linux. Assigning to @MatthijsKok

File: src/lancie-sponsors/lancie-sponsors.html [hunk 1]
```diff
@@ -42,7 +50,7 @@
       <div class="layout vertical flex-4">
         <h3>Premium sponsors</h3>
         <div class="layout horizontal wrap premium-sponsors">
-          <img src="/images-optimized/logos/sogeti.png" title="https://www.sogeti.nl/" />
+          <img src="/images-optimized/logos/sogeti.png" title="https://www.sogeti.nl/" id="sogeti" />
```
Comment (martijnjanssen) [inline src/lancie-sponsors/lancie-sponsors.html]: Please place the id as the first attribute on any html.

File: src/lancie-sponsors/lancie-sponsors.html [hunk 2]
```diff
@@ -12,17 +12,25 @@
...
-        object-fit: contain;
-        vertical-align: middle;
-        margin: 10px 6px;
+      .premium-sponsors img {
+        height: 100%;
       }
       
+      #sogeti {
+        position: relative;
+        top: 30px;
```
Comment (martijnjanssen) [inline src/lancie-sponsors/lancie-sponsors.html]: Do we really need sponsor specific styling?

**Comment by martijnjanssen** [comment]
Look good on my machine.

**Pull request merged**